models_logging/admin.py

```diff
@@ -68,13 +68,17 @@ def get_comment(self, obj):
         return '%s: %s' % (obj.action, obj.object_repr)
 
     def get_link_admin_object(self, obj):
-        if obj.object and obj.content_type.model_class() in admin.site._registry:
-            return format_html('<a href="%s">%s</a>' % (
-                reverse('admin:%s_%s_change' % (obj.content_type.app_label, obj.content_type.model),
-                        args=[obj.object_id]
-                ),
-                obj.object)
-            )
+        try:
+            if obj.object and obj.content_type.model_class() in admin.site._registry:
+                return format_html('<a href="%s">%s</a>' % (
+                    reverse('admin:%s_%s_change' % (obj.content_type.app_label, obj.content_type.model),
+                            args=[obj.object_id]
+                    ),
+                    obj.object)
+                )
+        except Exception as err:
+            print(err)
+            return None
```

Comment:
What is the purpose of this broad exception? Is there a known bug for this?

Reply:
Yes , this try in this code snippet is for when there is a null object. There are cases where, for example, a log of a specific model was generated and then that same model was deleted or renamed. So when you try to view the logs referring to that model/record it doesn't exist and there is an overflow on the screen to display.